src/components/Calculator.js

```diff
@@ -1,50 +1,44 @@
-        <Button btnId="clear" btnName="AC" handleClick={this.handleClick} />
-        <Button btnId="del" btnName="+/-" handleClick={this.handleClick} />
-        <Button btnId="del" btnName="%" handleClick={this.handleClick} />
-        <Button btnId="div" btnName="/" handleClick={this.handleClick} />
-        <Button btnId="seven" btnName="7" handleClick={this.handleClick} />
-        <Button btnId="eight" btnName="8" handleClick={this.handleClick} />
-        <Button btnId="nine" btnName="9" handleClick={this.handleClick} />
-        <Button btnId="mul" btnName="x" handleClick={this.handleClick} />
-        <Button btnId="four" btnName="4" handleClick={this.handleClick} />
-        <Button btnId="five" btnName="5" handleClick={this.handleClick} />
-        <Button btnId="six" btnName="6" handleClick={this.handleClick} />
-        <Button btnId="min" btnName="-" handleClick={this.handleClick} />
-        <Button btnId="one" btnName="1" handleClick={this.handleClick} />
-        <Button btnId="two" btnName="2" handleClick={this.handleClick} />
-        <Button btnId="three" btnName="3" handleClick={this.handleClick} />
-        <Button btnId="plus" btnName="+" handleClick={this.handleClick} />
-        <Button btnId="zero" btnName="0" handleClick={this.handleClick} />
-        <Button btnId="period" btnName="." handleClick={this.handleClick} />
-        <Button btnId="equal" btnName="=" handleClick={this.handleClick} />
-      </div>
-    );
-  }
-}
+  return (
+    <div className="calculator-wrapper">
+      <Display dId="display-view" total={total} next={next} operation={operation} />
+      <Button btnId="clear" btnName="AC" handleClick={handleClick} />
+      <Button btnId="del" btnName="+/-" handleClick={handleClick} />
+      <Button btnId="del" btnName="%" handleClick={handleClick} />
+      <Button btnId="div" btnName="/" handleClick={handleClick} />
+      <Button btnId="seven" btnName="7" handleClick={handleClick} />
+      <Button btnId="eight" btnName="8" handleClick={handleClick} />
+      <Button btnId="nine" btnName="9" handleClick={handleClick} />
+      <Button btnId="mul" btnName="x" handleClick={handleClick} />
+      <Button btnId="four" btnName="4" handleClick={handleClick} />
+      <Button btnId="five" btnName="5" handleClick={handleClick} />
+      <Button btnId="six" btnName="6" handleClick={handleClick} />
+      <Button btnId="min" btnName="-" handleClick={handleClick} />
+      <Button btnId="one" btnName="1" handleClick={handleClick} />
+      <Button btnId="two" btnName="2" handleClick={handleClick} />
+      <Button btnId="three" btnName="3" handleClick={handleClick} />
+      <Button btnId="plus" btnName="+" handleClick={handleClick} />
+      <Button btnId="zero" btnName="0" handleClick={handleClick} />
+      <Button btnId="period" btnName="." handleClick={handleClick} />
+      <Button btnId="equal" btnName="=" handleClick={handleClick} />
+    </div>
+  );
+};
```

Comment:
- [ ] Well done with the refactoring of your code. However, there is something here with certain actions like that of the memory of your calculator. If I want to perform a 5x5 multiplication. the memory is `55x `rather than `5x5 `even if the result is exact **25**. Please fix it :smile: 


### Step 1
![Screenshot from 2022-08-12 16-23-57](https://user-images.githubusercontent.com/6615394/184403744-9375499f-1a60-4f5d-941c-753b806d580d.png)

### Step 2
![Screenshot from 2022-08-12 16-24-04](https://user-images.githubusercontent.com/6615394/184403741-7e562dc3-fbcc-4692-92b0-13ede4f52c38.png)
### Step 3
![Screenshot from 2022-08-12 16-24-24](https://user-images.githubusercontent.com/6615394/184403739-d15d1f09-bf25-4e24-9c2c-fec8cc4fd52e.png)
### Result
![Screenshot from 2022-08-12 16-24-36](https://user-images.githubusercontent.com/6615394/184403734-70252865-d8c2-481f-946a-33c38d1e6e24.png)